wagtail/models/i18n.py

```diff
@@ -474,7 +474,7 @@ def set_locale_on_new_instance(sender, instance, **kwargs):
         return
 
     # If this is a fixture load, use the global default Locale
-    # as the page tree is probably in an flux
+    # as the page tree is probably in a flux
```

Comment:
"in flux" is the correct phrase.
```suggestion
    # as the page tree is probably in flux
```

Reply:
Thanks a lot for the correction.
I'll change it immediately.